src/Form/Type/PhoneNumberType.php

```diff
@@ -128,6 +132,7 @@ public function configureOptions(OptionsResolver $resolver)
             'by_reference' => false,
             'error_bubbling' => false,
             'country_choices' => [],
+            'country_display_type' => self::DISPLAY_COUNTRY_FULL,
```

Comment:
This looks like a break, we currently use short format, isn't it?

Reply:
nop, currently we use "France (+33)" the "FULL" format

Reply:
my bad